openedx/features/learner_profile/views/learner_achievements.py

```diff
@@ -44,7 +44,8 @@ def _get_ordered_certificates_for_user(self, request, username):
                     course_overview = CourseOverview.get_from_id(course_key)
                     course_certificate['course'] = course_overview
                     if certificates_viewable_for_course(course_overview):
-                        passing_certificates.append(course_certificate)
+                        if course_certificate['is_pdf_certificate'] or course_overview.has_any_active_web_certificate:
```

Comment:
Does this have to be a nested if? Can it be simplified at all?

Reply:
@fysheets We cannot use nested if here otherwise it will not append certificate for Honor/PDF certificates.

Reply:
Added an explanation comment for this.